brd-android/fabriik-common/src/test/java/com/fabriik/common/utils/FlagUtilTest.kt

```diff
@@ -0,0 +1,264 @@
+package com.fabriik.common.utils
+
+import com.fabriik.common.R
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class FlagUtilTest(private val countryCode: String, private val expectedDrawableId: Int) {
```

Comment:
who does create and call this class?

Reply:
This is the way how parameterized tests work. `getParams()` defines all test cases (inputs and expected results) and a test runner use them to create FlagUtilTest class for every test case.

Reply:
fair enough